contracts/ethereum/SnapshotXZodiacModule/ProposalRelayer.sol

```diff
@@ -1,73 +1,79 @@
-//SPDX-License-Identifier: UNLICENSED
+//SPDX-License-Identifier: MIT
 
 pragma solidity ^0.8.6;
 
 import '@gnosis.pm/zodiac/contracts/guard/Guardable.sol';
+import './Interfaces/IStarknetCore.sol';
 
-interface IStarknetCore {
-  /**
-    Consumes a message that was sent from an L2 contract.
-    Returns the hash of the message.
-  */
-  function consumeMessageFromL2(uint256 fromAddress, uint256[] calldata payload)
-    external
-    returns (bytes32);
-
-  function l2ToL1Messages(bytes32 msgHash) external view returns (uint256);
-}
-
+/**
+ * @title Snapshot X L1 Proposal Relayer
+ * @author @Orland0x - <orlandothefraser@gmail.com>
+ * @dev Work in progress
+ */
 contract SnapshotXProposalRelayer is Guardable {
-  // The StarkNet core contract.
+  /// The StarkNet Core contract
   IStarknetCore public starknetCore;
 
-  //address of the L2 end of the finalized proposal message bridge
+  /// Address of the starknet contract that will send execution details to this contract in a L2 -> L1 message
```

Comment:
Any reason to use "///" instead of "//"? Also we should write "StarkNet" with the official casing and be consistent. You should not write it differentlty on each comments. Now i see 3 different casing: StarkNet starknet and Starknet in the function comments

Reply:
from the natspec docs: https://docs.soliditylang.org/en/develop/natspec-format.html
"For Solidity you may choose /// for single or multi-line comments, or /** and ending with */."